ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -305,7 +309,7 @@ private Long incrementalDump(Path dumpRoot, DumpMetaData dmd, Path cmRoot, Hive
       NotificationEvent ev = evIter.next();
       lastReplId = ev.getEventId();
       Path evRoot = new Path(dumpRoot, String.valueOf(lastReplId));
-      dumpEvent(ev, evRoot, cmRoot, hiveDb);
+      dumpEvent(ev, evRoot, dumpRoot, cmRoot, hiveDb);
```

Comment:
Why is this dumpRoot and not hiveDumpRoot

Reply:
hiveDumpRoot is recieved as dumpRoot in the current method. I haven't renamed existing parameter names.